be/src/runtime/snapshot_loader.cpp

```diff
@@ -756,10 +756,17 @@ bool SnapshotLoader::_end_with(const std::string& str, const std::string& match)
     return false;
 }
 
+bool SnapshotLoader::_contains(const std::string& str, const std::string& match) {
+    if (str.size() >= match.size() && str.find(match) != std::string::npos) {
+        return true;
+    }
+    return false;
+}
+
 bool SnapshotLoader::_is_index_files(const std::string& str) {
     return _end_with(str, "fdt") || _end_with(str, "fdx") || _end_with(str, "fnm") || _end_with(str, "frq") ||
            _end_with(str, "nrm") || _end_with(str, "prx") || _end_with(str, "tii") || _end_with(str, "tis") ||
-           _end_with(str, "null_bitmap") || _end_with(str, "segments_2") || _end_with(str, "segments.gen");
+           _end_with(str, "null_bitmap") || _contains(str, "segments");
```

Comment:
Why segments is the prefix of index?

Reply:
This is the index file name written by CLucene lib

Reply:
Can we modify this name through the interface?